miio/integrations/yeelight/__init__.py

```diff
@@ -265,13 +265,17 @@ def __init__(
         debug: int = 0,
         lazy_discover: bool = True,
         model: str = None,
+        light_type: YeelightSubLightType = YeelightSubLightType.Main,  # for now only Main support this is for future PR.
```

Comment:
```suggestion
```
Please remove this for now, we don't want to extend the constructor signature for any more special cases.